README.md

```diff
@@ -41,7 +44,7 @@ and package Skript for you:
 gradlew clean build # on Windows
 ```
 
-You can get source code from the [releases](https://github.com/SkriptLang/Skript/releases) page.
+You can get source code from the [Releases](https://github.com/SkriptLang/Skript/releases) page.

```

Comment:
It is capitalized on pages only because it is part of title.

Reply:
Then "releases page" should be link, not just "releases"?

Reply:
Releases page, as in a page that contains releases sounds better to me. Is it not grammatically correct?